go/vt/sqlparser/sql.y

```diff
@@ -649,6 +649,11 @@ command:
 | prepare_statement
 | execute_statement
 | deallocate_statement
+| CHECKSUM TABLE table_name_list
+  {
+    $$ = &ChecksumExpr{Tables: $3}
+  }
+
```

Comment:
Can we move this into its own `checksum_statement` expression? It also doesn't seem to support the full syntax, see https://dev.mysql.com/doc/refman/8.0/en/checksum-table.html.

There's the `QUICK` or `EXTENDED` options at the end that we don't seem to parse here?